src/containers/bookform.js

```diff
@@ -0,0 +1,143 @@
+import { connect } from 'react-redux';
+import React, { useState } from 'react';
+import { addBook } from '../actions/BookActions';
+import categories from '../utils/cat';
+import changeCategory from '../actions/categoryActions';
+
+const bookForm = (props) => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const [pages, setPages] = useState('');
+  const [category, setCategory] = useState('');
+  const [pagesread, setPagesread] = useState('');
+  const [chapter, setChapter] = useState('');
+  const [chapterName, setChapterName] = useState('');
```

Comment:
- [ ] Kindly delete these and ensure that you use redux to handle all the states